spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -461,6 +469,13 @@ contract PerpetualTranche is
         return r;
     }
 
+    /// @inheritdoc IPerpetualTranche
+    /// @dev CAUTION: Only the whitelisted vault can call this function. The logic controlling the frequency
+    ///      and magnitude of debasement should be thoroughly vetted.
+    function debase(uint256 perpAmtMint) external override onlyVault afterStateUpdate nonReentrant whenNotPaused {
```

Comment:
A better alignment with interface / component ownership would be to have this function take in an `underlying` amount that should be transferred. Let perp be the authority for how it makes that happen.

Instead of a function called debase, I'd rather keep it neutral (inline with the two-sided view of the system comment). We could call this "balance" or "rebalance to vault", or similar. But probably best to come back to this after we figure out the other items.